e2e/process-services/process-attachment-list-action-menu.e2e.ts

```diff
@@ -133,6 +133,8 @@ describe('Attachment list action menu for processes', () => {
         await processFiltersPage.selectFromProcessList(processName.completed);
         await processDetailsPage.checkProcessTitleIsDisplayed();
 
+        await browser.sleep(1000);
```

Comment:
Why do we need browser.sleep here? Is it absolutely necessary?

Reply:
I have tried other fixes, but only this one helped. 
I can raise a ticket to improve this test later.
Leave this workaround or remove it and exclude the test.

Reply:
I see often:
  await processDetailsPage.checkProcessTitleIsDisplayed();
  used with:
await processFiltersPage.waitForTableBody();
did you try it?